frontend/website/src/pages/LeaderboardPage.re

```diff
@@ -219,14 +219,25 @@ let make = (~lastManualUpdatedDate) => {
     dispatch(UsernameEntered(username));
   };
 
+  let releaseInfo = "Release 3.2a";
+  let challengeInfo = [|
+    (None, "Coda Sandbox", Some(1000)),
+    (Some("**"), "Share Your Thoughts", None),
+    (Some("**"), "Tell a Friend", Some(1000)),
+    (None, "Megaphone", Some(1000)),
+    (Some("**"), "Community Builder", None),
+    (None, "Let's Meetup", Some(1000)),
+    (None, "Bonus", Some(1000)),
+  |];
+
   <Page title="Testnet Leaderboard">
     <Wrapped>
       <div className=Styles.page> <Summary lastManualUpdatedDate /> </div>
       <div className=Styles.filters>
-        <SearchBar onUsernameEntered username={state.username} />
-        <ToggleButtons currentToggle={state.currentToggle} onTogglePress />
-        <FilterDropdown currentFilter={state.currentFilter} onFilterPress />
-      </div>
+        //<SearchBar onUsernameEntered username={state.username} />
+        //<ToggleButtons currentToggle={state.currentToggle} onTogglePress />
+        //<FilterDropdown currentFilter={state.currentFilter} onFilterPress />
+         <ChallengePointsTable releaseInfo challengeInfo /> </div>
```

Comment:
This is done just for testing purposes. When this PR is ready to land, the comments will be removed and the `ChallengePointsTable` component will be rendered on the correct page.